lib/android/examples/src/ui_test.nit

```diff
@@ -20,44 +20,37 @@ module ui_test is
 	app_version(0, 1, git_revision)
 	app_namespace "org.nitlanguage.ui_test"
 	android_manifest_activity """android:theme="@android:style/Theme.Light""""
```

Comment:
You felt like reviewing unmodified line of code today?

I'm not convinced, I prefer the first version to the second. But would it make you (and your grammar) happy with extra spaces? 
~~~
"""android:theme="@android:style/Theme.Light""""
"android:theme=\"@android:style/Theme.Light\""
""" android:theme="@android:style/Theme.Light" """
~~~